ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_linalg.py

```diff
@@ -285,3 +285,71 @@ def test_eig(
         test_values=False,
         x=x[0],
     )
+
+
+# multi_dot
+@st.composite
+def _generate_multi_dot_dtype_and_arrays(draw):
+    input_dtype = [draw(
+        st.sampled_from(draw(helpers.get_dtypes("numeric")))
+    )]
+    matrices_dims = draw(
+        st.lists(st.integers(min_value=2, max_value=10), min_size=4, max_size=4)
+    )
+    shape_1 = (matrices_dims[0], matrices_dims[1])
+    shape_2 = (matrices_dims[1], matrices_dims[2])
+    shape_3 = (matrices_dims[2], matrices_dims[3])
+    
+    matrix_1 = draw(
+        helpers.dtype_and_values(
+            shape=shape_1,
+            dtype=input_dtype,
+            min_value=-10,
+            max_value=10,
+        )
+    )
+    matrix_2 = draw(
+        helpers.dtype_and_values(
+            shape=shape_2,
+            dtype=input_dtype,
+            min_value=-10,
+            max_value=10,
+        )
+    )
+    matrix_3 = draw(
+        helpers.dtype_and_values(
+            shape=shape_3,
+            dtype=input_dtype,
+            min_value=-10,
+            max_value=10,
+        )
+    )
+
+    return input_dtype, (matrix_1[1][0], matrix_2[1][0], matrix_3[1][0])
+
+
+@handle_test(
+    fn_tree="functional.ivy.experimental.multi_dot",
+    dtype_x=_generate_multi_dot_dtype_and_arrays(),
+    test_gradients=st.just(False),
+)
+def test_multi_dot(
+    dtype_x,
+    test_flags,
+    backend_fw,
+    fn_name,
+    ground_truth_backend,
+):
+    dtype, x = dtype_x
+    # test_flags.instance_method = False
```

Comment:
If setting the instance method flag to False is a quick fix, the error may be in the array instance method or in the container one. Have you tried to use the debugger to find where the exception is exactly happening? 🤔